### ISSUE TITLE
%include breaks if space is included in path

### ISSUE THREAD

**Issue by tkdchen** [rpm-software-management/rpm#125]
``rpm-4.13.0-1.fc24.x86_64``

Steps to replicate

```bash
$ cd my\ package/
$ git clone https://src.fedoraproject.org/git/rpms/systemd.git
$ cd systemd
$ python
>>> import rpm
>>> rpm.expandMacro('%define _sourcedir "/home/cqi/my package/systemd/"')
''
>>> rpm.spec('systemd.spec')
error: systemd.spec:367: malformed %include statement
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: can't parse specfile
```

**Comment by Conan-Kudo** [comment]
Does doing the following work?

```python
rpm.expandMacro('%define _sourcedir "/home/cqi/my\ package/systemd/"')
```

**Comment by tkdchen** [comment]
@Conan-Kudo 

No. It doesn't work. Same error.

**Comment by pmatilai** [comment]
Pavlina, please have a look at this.

**Comment by pmatilai** [comment]
Fixed in commit 9d38b2f6d92ef982db2488462619a780c9b87973
